client/src/js/components/App.jsx

```diff
@@ -1,22 +1,43 @@
 import React from 'react';
+import axios from 'axios';
+import Header from './Header';
+import Accomodations from './Accomodations';
+import '../../../css/main.css';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {
+    this.state = {};
+  }
 
-    };
+  componentDidMount(id) {
+    axios.get(`http://127.0.0.1:3001/homes/${1058}`)
```

Comment:
might want to put this in a helper function which is independent of components. you will want to reuse it in other places